### PR TITLE
Fairness Overview

### PR DESCRIPTION
**Pull request by Hilly12**
Add an overview for fairness to the documentation

### CODE REVIEW

File: docs/user_guide/fairness.rst [hunk 1]
```diff
@@ -1,125 +1,49 @@
-Fairness
+Overview
 ========
 
-This is a general guide to assessing fairness in supervised learning tasks (classification, regression)
-using structural datasets.
-
 Literature Review
 -----------------
```
Comment (rob-tay) [inline docs/user_guide/fairness.rst:6]: I would remove the "Literature Review" heading

File: docs/user_guide/fairness.rst [hunk 2]
```diff
@@ -1,125 +1,49 @@
...
+.. math::
 
-However, if the size of the data is limited and some groups are underrepresented, the model might unintentionally produce
-unfair results based on the local observations of the sample distributions.
+  P(R \mid A = a) = P(R \mid A = b)\quad \forall a,b \in A
 
-Negative Legacy
-^^^^^^^^^^^^^^^
+While Barocas et. al propose 2 alternative fairness or non-discrimination criteria, separation and sufficiency,
+which are applicable to a range of problems, we have chosen to work with independence because of its generality [2]_.
```
Comment (rob-tay) [inline docs/user_guide/fairness.rst:38]: Can you keep the reference style consistent? There's a mix of [x] and "author name et al."

File: docs/user_guide/fairness.rst [hunk 3]
```diff
@@ -1,125 +1,49 @@
...
+Working with this abstract definition of fairness, we quantify the bias of a variable :math:`T` in a group :math:`a`,
+as the statistical distance between the the probability distributions of :math:`P(T \mid A = a)` and :math:`P(T)`.
```
Comment (rob-tay) [inline docs/user_guide/fairness.rst:40]: I think after this section we need to give more details, and how the features of fairlens relate to this

**Review by rob-tay — CHANGES_REQUESTED** [review]
Nice thanks! I think before merging this needs to be linked back to the features of FairLens.

File: docs/user_guide/fairness.rst (hunk 3 above)
Comment (Hilly12) [reply docs/user_guide/fairness.rst:40]: Adding another section for this.

File: docs/user_guide/fairness.rst (hunk 2 above)
Comment (Hilly12) [reply docs/user_guide/fairness.rst:38]: Forgot a [1]_ here. Would it be consistent to say "While Barocas et. al [1] propose...".

Comment (rob-tay) [reply docs/user_guide/fairness.rst:38]: Yeah I think thats ok, alternatively can do just "While [1] propose..."

**Review by rob-tay — APPROVED** [review]
I think we can add more to this in the future, but looks good for now! Thanks 👍

**Pull request merged**